dev/docker/trino/conf/update-trino-conf.sh

```diff
@@ -13,3 +13,10 @@ sed "s/GRAVITINO_HOST_IP:GRAVITINO_HOST_PORT/${GRAVITINO_HOST_IP}:${GRAVITINO_HO
 # Update `gravitino.metalake = GRAVITINO_METALAKE_NAME` in the `conf/catalog/gravitino.properties`
 sed "s/GRAVITINO_METALAKE_NAME/${GRAVITINO_METALAKE_NAME}/g" "${trino_conf_dir}/catalog/gravitino.properties.tmp" > "${trino_conf_dir}/catalog/gravitino.properties"
 rm "${trino_conf_dir}/catalog/gravitino.properties.tmp"
+
+# Check the number of Gravitino connector plugins present in the Trino plugin directory
+num_of_gravitino_connector=$(ls /usr/lib/trino/plugin/gravitino | grep gravitino-trino-connector-* | wc -l)
```

Comment:
Only check `gravitino-trino-connector-*` seems not enough, each jar from Gravitino(start with `gravitino-`) needs to be checked, such as `gravitino-bundle-catalog-*` and so on.

![image](https://github.com/datastrato/gravitino/assets/24897598/437bd798-e482-4237-befd-03f587f583ae)

Reply:
If we apply this scenario, the situation becomes complex. For example, If we update the version of `bootstrap` from 235 and `236`, do we need to check the version of it? Furthermore, there would be more modules that start with `gravitino`, it's very hard to maintain them. So, I will only check the version of `trino-connector` here.

Reply:
However, the current approach does not seem to address the issue of Trino integration test failures caused by inconsistencies between the actual code and the running code version during local Trino integration test runs.

Reply:
Or we can offer how to do then in the error message?

Reply:
BTW, I think that this issue is caused by Trino's integration testing mechanism and should not be the responsibility of the user to resolve. Instead, the testing mechanism should be improved to prevent this issue.

Reply:
@mchades 
I added a check in the build script regarding the version. Please take a look.

Reply:
@diqiu50 
Should we still have the check here as we had added similar logic in the `build.gradle.kts`?